validator/test/fixtures/storage_bucket_policy_only/assets/data.json

```diff
@@ -0,0 +1,134 @@
+[{
```

Comment:
we've also changed the location of the test fixtures/assets. PTAL at https://github.com/forseti-security/policy-library/pull/82

Reply:
noted the reorganization that happened yesterday, is there anything I should change for this file? I've referenced the structure of storage_logging folder.